src/MapViewController.swift

```diff
@@ -17,12 +17,12 @@ import OnTheRoad
   case GeneralError, AnnotationDoesNotExist, APIKeyNotSet, RouteDoesNotExist
 }
 
-public class MapViewController: TGMapViewController, LocationManagerDelegate, TGRecognizerDelegate {
+public class MapViewController: UIViewController, LocationManagerDelegate, TGRecognizerDelegate {
 
   //Error Domains for NSError Appeasement
   public static let MapzenGeneralErrorDomain = "MapzenGeneralErrorDomain"
-
-
+  
+  var tgViewController: TGMapViewController = TGMapViewController()
```

Comment:
I might be missing something but isn't that the intent of this PR? If we mark it as public it might be creating ambiguous access to the method of the Tangram controller, since someone would have two ways to interact the map view, through this accessor and through the wrapped methods.